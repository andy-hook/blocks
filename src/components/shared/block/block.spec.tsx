import React from "react"
import Block from "./block"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<Block />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <Block />)
    expect(tree).toMatchSnapshot()
  })
})
