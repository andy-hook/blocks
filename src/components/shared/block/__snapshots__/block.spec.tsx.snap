// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Block /> renders correctly 1`] = `
<article
  className="sc-iRbamj blSEcj"
>
  <div
    className="sc-gzVnrw cnjFid sc-jTzLTM iziihJ"
  >
    <a
      className="sc-iwsKbI Rjfvb"
      href="/"
      type="tertiary"
    >
      <div
        className="sc-gZMcBi irilpd"
      >
        <div
          className="sc-htoDjs hZurrr sc-VigVT eJzOxy"
        >
          <svg
            className="sc-dnqmqq iydQTw"
          >
            <use
              xlinkHref="/icon-sprite.svg#arrow-left"
            />
          </svg>
        </div>
        <span
          className="sc-gqjmRU krxleO"
        >
          All Blocks
        </span>
      </div>
    </a>
    <header
      className="sc-fjdhpX wAbRy"
    >
      <div
        className="sc-cSHVUG hRXcfM"
      >
        <div
          className="sc-htpNat fRItwi sc-jzJRlG kpJqLT"
          size="lg"
        >
          <div
            className="sc-bxivhb kzfWct"
          >
             
          </div>
        </div>
        <div
          className="sc-bdVaJa iZkTwA"
          size="lg"
        >
          <div
            className="sc-bwzfXH ePBugQ"
          >
             
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX cFlEyZ sc-kAzzGY dBGQmE"
      >
        <button
          className="sc-EHOje klPAPL"
        >
          <div
            className="sc-bZQynM GNSbg"
          >
            Details
          </div>
        </button>
        <button
          className="sc-EHOje fbAsfT"
        >
          <div
            className="sc-bZQynM GNSbg"
          >
            Transactions
          </div>
        </button>
      </div>
    </header>
  </div>
  <div
    className="sc-gzVnrw cnjFid sc-kGXeez jrWVKz"
  >
    <ul
      className="sc-kpOJdX kAzjOl"
    >
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Size
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb dAzbVZ"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Timestamp
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb kzfWct"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Miner
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb kzfWct"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Difficulty
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb kzfWct"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Total Difficulty
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb drrWxJ"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Gas Used
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb kzfWct"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gfSCZK sc-dxgOiQ cdhNWs"
      >
        <div
          className="sc-bdVaJa ceXrEC"
          size="sm"
        >
          Gas Limit
        </div>
        <div
          className="sc-kgoBCf donpaw"
        >
          <div
            className="sc-htpNat leGpXE"
            size="lg"
          >
            <div
              className="sc-bxivhb dAzbVZ"
            >
               
            </div>
          </div>
        </div>
      </div>
    </ul>
  </div>
</article>
`;
